user/urls.py

```diff
@@ -1,10 +1,17 @@
 from django.urls import path
+from rest_framework_simplejwt.views import (TokenRefreshView,
+                                            TokenObtainPairView,
+                                            TokenVerifyView)
+
 from user.views import CreateUserView, CreateTokenView, ManageUserView
```

Comment:
Remove CreateTokenView import as you use JWT auth

Reply:
Removed